src/components/Shared/Navbar/index.jsx

```diff
@@ -5,20 +5,63 @@ import {
   LOGO_ICON_ALT_TEXT,
   ICON_SEARCH,
   SEARCH_ICON_ALT_TEXT,
+  PAGINATION_LIMIT,
 } from "utils/constant";
+import PropTypes from "prop-types";
+import { useDispatch } from "react-redux";
+import { loadingState, paginationLimitUpdate, searchTask } from "store/actions";
+import { sanitizeText } from "utils/helpers/sanitizeText";
+import { debounce } from "utils/helpers/debouce";
+
+const Navbar = ({ isSearchBarVisible, onSearchBarVisible }) => {
+  const dispatch = useDispatch();
+
+  const handleInputChange = (event) => {
```

Comment:
can we use regular function syntax instead of arrow function ? [reference](https://github.com/azo-wtag/todo-app-react/pull/4#discussion_r1172078490)
```suggestion
  function handleInputChange (event) {
```